package.json

```diff
@@ -8,7 +8,7 @@
     "lib/install/config/webpacker.yml"
   ],
   "engines": {
-    "node": ">= 12.13.0 || >=14",
+    "node": "^12.13.0 || ^14 || >=16",
```

Comment:
Why not that?

```suggestion
    "node": "^12.13.0 || ^14 || ^16",
```

Reply:
That would not allow users to use version 17 of node.js.